Taskfile.yml

```
@@ -327,6 +327,11 @@ tasks:
     cmds:
       - poetry run mkdocs serve
 
+  get-version:
```

```suggestion
  general:get-version:
```

Although the Arduino CLI taskfile does not fully comply due to legacy, the standardized style for the Tooling taskfiles is to use a namespace prefix on all task names other than the convenience "umbrella" tasks (`build`, `check`, `fix`).